kuma/javascript/src/toc.jsx

```diff
@@ -48,11 +59,20 @@ export default function TOC({ html }: Props) {
 
     return (
         <aside className="document-toc-container">
-            <section className="document-toc" ref={tocRef}>
+            <section className="document-toc" ref={documentTOCRef}>
```

Comment:
Sorry for nitting this here, given that the code existed before this PR, but I think instead of using ref&effect we should do
```jsx
<section onClick={sendTOCClicks}/>
```
Should do the same thing, with less code in this context, no?